Assets/MRTK/Providers/UnityAR/MixedReality.Toolkit.Providers.UnityAR.nuspec

```diff
@@ -24,9 +24,9 @@
     <file src="**" exclude="*.nuspec;*.nuspec.meta;*.props;*.props.meta;*.targets;*.targets.meta" target="MRTK\" />
 
     <!-- Reuse MixedReality.Toolkit.targets, as the MSBuild logic is the same for all MRTK nuget packages. -->
-    <file src="..\..\MixedRealityToolkit\MixedReality.Toolkit.targets" target="build\Microsoft.MixedReality.Toolkit.Providers.UnityAR.targets" />
+    <file src="..\..\..\MixedRealityToolkit\MixedReality.Toolkit.targets" target="build\Microsoft.MixedReality.Toolkit.Providers.UnityAR.targets" />
```

Comment:
We'll have to remember to update this again when the core folder is renamed

Reply:
Is this something you might want to add a code validation step for?